Modules/Microsoft365DSC/DSCResources/MSFT_SCDeviceTenantPolicy/MSFT_SCDeviceTenantPolicy.psm1

```diff
@@ -0,0 +1,395 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter()]
+        [System.String]
+        $Guid,
```

Comment:
Since this is the key, it is required for the property to be marked as mandatory. Otherwise Start-DSCConfiguration will fail.

Reply:
@sasistla, could you please fix this last comment today? Then we can include it in the next release.